### PR TITLE
circular percentage chart for levels

### PR DESCRIPTION
**Pull request by Argn0**
https://deploy-preview-2718--opendota-staging.netlify.app/matches/5756960669

### CODE REVIEW

**Comment by howardchung** [comment]
Nice! Think it could use a little more padding between number and circle.

Also this might be a bit confusing as in game it shows the amount of exp needed to get to the next level

**Comment by Argn0** [comment]
I've slightly increased the padding.
I'm afraid I can't think of a solution to solve the second issue, but I assume people figure it out quickly.

**Comment by howardchung** [comment]
Also it doesn't look quite centered (too far to the right?)

File: src/components/Match/StyledMatch.jsx [hunk 1]
```diff
@@ -68,6 +68,31 @@ export const StyledAghanimsBuffs = styled.div`
   }
 `
 
+export const StyledLevel = styled.div`
+  position: relative;
+  left: 2px;
+
+  .level_number {
+    position: absolute;
+    top: 6px;
```
Comment (howardchung) [inline src/components/Match/StyledMatch.jsx:77]: would be cool if we can center it automatically with properties rather than pixel values

Comment (Argn0) [reply src/components/Match/StyledMatch.jsx:77]: I'll try but I'm not sure if it's possible.

**Comment by Argn0** [comment]
If there is still something off just let me know.

**Comment by Argn0** [comment]
OK, I think this is as good as it gets. There was an issue with the table headers that prevented me from centering properly.

**Review by howardchung — APPROVED** [review]

**Pull request merged**